CONTRIBUTING.md

```diff
@@ -3,29 +3,24 @@ the process of getting changes merged in and released.
 
 1. [Setting up Marionette locally](#setting-up-marionette-locally)
 2. [Reporting a bug](#reporting-a-bug)
-  1. [When you have a patch](#when-you-have-a-patch)
-  2. [When you don't have a bug fix](#when-you-dont-have-a-bug-fix)
-3. [Running Tests](#running-tests)
-4. [Updating docs](#updating-docs)
+3. [Submitting patches and fixes](#submitting-patches-and-fixes)
+4. [Running Tests](#running-tests)
 
```

Comment:
I wanted to stick to the top-level headers here so the reader could quickly find the thing they're currently working on e.g. they're submitting a pull request.

That said, I can definitely see the arguments for including them as someone might be thinking "what do I do for docs!"